Nelson Mosquera/liveserver/package.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "liveserver",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "live": "npm install livereload --save & npm install http-server & node livereload & node http-server"
```

```suggestion
    "live": "node livereload"
```
`npm live` doesn't need to install the dependencies everytime the command is executed, you should assume the developer already executed `npm i` before running any other command in your app.